editoast/src/views/train_schedule.rs

```diff
@@ -524,13 +548,53 @@ pub async fn train_simulation_batch(
         .collect())
 }
 
+/// Compute in batch the simulation of a list of train schedule
+///
+/// Note: The order of the returned simulations is the same as the order of the train schedules.
+pub async fn train_simulation_batch(
```

Comment:
The `train_simulation_batch` should simply build the PhysicsConsistParameters from the rolling stock names then call the `consist_train_simulation_batch`